### DISCUSSION TITLE
How to have log into a file?

### DISCUSSION THREAD

**Discussion by ArchieCoder** [unoplatform/uno.extensions#2477] [Show and tell]
In the default Uno template App.xaml.cs, there is the following line:

**.UseSerilog(consoleLoggingEnabled: true, fileLoggingEnabled: true)**

The flag fileLoggingEnabled=true does not seem to do anything. Is it a bug or is there a configuration missing?

**Comment by ArchieCoder** [comment]
Looking at the decompiled code, is it possible that NETSTANDARD is not defined at all anymore?

![image](https://github.com/user-attachments/assets/96768741-66f4-4d68-8e9b-e67945340aef)

**Comment by ArchieCoder** [comment]
I copied the HostBuilderExtensions.cs file into my project and I commented out the line with NETSTANDARD.

The GetLogFilePath will return the filename: "ElementsTechnician, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null.log" This might be another bug.

**Comment by nickrandolph** [comment]
@ArchieCoder thanks for picking up on this. We've got a few updates coming (eg https://github.com/unoplatform/uno.extensions/pull/2437), so we'll try to get this resolved after that's merged. I'll convert this to an issue in the meantime so we can track it

**Comment by ArchieCoder** [comment]
Big update!! Closing the issue for now since I unblocked myself today. I will update my code if things change later. Thanks.
